### PR TITLE
Add descriptions from README to Options type

### PR DESCRIPTION
**Pull request by qm3ster**
This will make them available in editor.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -14,12 +14,30 @@ import { validate } from './validator'
 
 export { EnumJSONSchema, JSONSchema, NamedEnumJSONSchema } from './types/JSONSchema'
 
-export interface Options {
+export interface Options export interface Options {
```
Comment (bcherny) [inline src/index.ts]: You might want to double check this.

**Comment by bcherny** [comment]
Closing and re-opening didn't seem to do it. @qm3ster do you mind rebasing off the latest master?

**Comment by qm3ster** [comment]
Terribly sorry, that's gone now. (rebased too)

**Comment by qm3ster** [comment]
I can't seem to retrigger it, but I ran the tests locally.
The punctuation issues is twofold:
 1. These are not full sentences, just complex nouns for props and verbs for bools
 2. I based it on what I see most libraries provide

**Comment by bcherny** [comment]
@qm3ster Hence why it's a nit. We use this style elsewhere, and I'd like to keep it consistent.

**Pull request merged**